tools/src/commands/ClientInstall.ts

```diff
@@ -1,91 +1,131 @@
-
-  const installResult = await Simulator.installExpoOnSimulatorAsync({ url: clientUrl, simulator });
-
-  if (installResult.status !== 0) {
-    throw new Error('Installing Expo client simulator failed!');
+  const simulator = await Simulator.queryFirstBootedSimulatorAsync();
+  if (!simulator) {
+    throw new Error('No booted iOS simulator found.');
   }
 
-  const appIdentifier = 'host.exp.Exponent';
+  console.log('Uninstalling previously installed Expo Go...');
 
-  console.log(`Launching Expo client with identifier ${chalk.blue(appIdentifier)}...`);
+  await Simulator.uninstallAppFromSimulatorAsync(simulator, EXPO_GO_APP_ID_IOS);
 
-  await spawnAsync('xcrun', ['simctl', 'launch', 'booted', appIdentifier]);
+  console.log(`Downloading Expo Go from ${chalk.blue(downloadUrl)}`);
+  const tmpDir = await fs.promises.mkdtemp(path.join(os.tmpdir(), 'expotools-client-install-'));
+  try {
+    const downloadFilePath = await downloadExpoGoAsync({ downloadUrl, targetDir: tmpDir });
+    const appPath = path.join(tmpDir, 'Expo Go.app');
+    await fs.promises.mkdir(appPath, { recursive: true });
+    await tarExtract({
+      file: downloadFilePath,
+      cwd: appPath,
+      strip: 1,
+    });
+    console.log(`Extracted to ${chalk.blue(tmpDir)}`);
+    console.log(`Installing Expo Go from ${chalk.blue(appPath)} on iOS simulator...`);
+    await Simulator.installSimulatorAppAsync(simulator, appPath);
+    console.log(`Launching Expo Go with identifier ${chalk.blue(EXPO_GO_APP_ID_IOS)}...`);
+    await Simulator.launchSimulatorAppAsync(simulator, EXPO_GO_APP_ID_IOS);
+  } catch (error: unknown) {
+    if (error instanceof Error) {
+      console.error(chalk.red(`Unable to install Expo Go: ${error.message}`));
+    }
+  } finally {
+    await fs.promises.rm(tmpDir, { recursive: true, force: true });
+  }
 }
 
-async function downloadAndInstallOnAndroidAsync(clientUrl: string): Promise<void> {
+async function downloadAndInstallOnAndroidAsync(downloadUrl: string): Promise<void> {
+  const tmpDir = await fs.promises.mkdtemp(path.join(os.tmpdir(), 'expotools-client-install-'));
   try {
     console.log('Checking if the are any Android devices or emulators connected...');
 
-    const devices = await Android.getAttachedDevicesAsync();
+    const devices = await AndroidDevice.getAttachedDevicesAsync();
     if (devices.length === 0) {
       throw new Error('No connected devices or emulators found.');
     }
 
     const device = devices[0];
     if (devices.length > 1) {
       console.log(
-        `More than one Android device found. Installing on the first one found, ${device.name}.`
+        `More than one Android device found. Installing on the first one found, ${device}.`
       );
     }
 
-    if (!device.isAuthorized) {
-      throw new Error(
-        `This computer is not authorized for developing on ${device.name}. See https://expo.fyi/authorize-android-device.`
-      );
-    }
```

Comment:
i don't check unauthorized devices anymore. people should setup authorization beforehand